src/apps/material/material.entry.tsx

```diff
@@ -77,6 +77,8 @@ interface MaterialEntryTextProps {
   modalReservationFormSmsInputFieldDescriptionText: string;
   modalReservationFormSmsInputFieldLabelText: string;
   numberDescriptionText: string;
+  addToFavoritesAriaLabelText: string;
+  removeFromFavoritesAriaLabelText: string;
```

Comment:
Please put them in alphabetical order. Since we have so many texts and we need to use them in dpl-cms also, having them sorted alphabetically helps managing them :)